tests/test_serp.py

```diff
@@ -327,15 +327,15 @@ def test_metadata():
                     "title": "Results Per Page",
                 },
                 "item_type": {
-                    "anyOf": [{"type": "string"}, {"type": "null"}],
-                    "default": None,
+                    "default": "nothing",
                     "description": (
                         "If specified, follow organic search result links, "
                         "and extract the selected data type from the target "
                         "pages. Spider output items will be of the specified "
                         "data type, not search engine results page items."
                     ),
                     "enum": [
+                        "nothing",
```

Comment:
It seems someone could get confused - they read "follow and extract nothing", and think what the hell the spider is going to do if it's extracting nothig? I'm not sure what's the better alternative though. Any ideas?

Reply:
We could replace “nothing” with “only_extract_serp” or “only_extract”.

Alternatively, we could name the option only “Extract”, replace “nothing” with “serp”, and consider the fact that following is needed for other types an implementation detail.

Reply:
> We could replace “nothing” with “only_extract_serp” or “only_extract”.

This may work, but it also doesn't communicatd that the "follow" step is not applied. "only extract" is also a bit confusing, because it's not "extract" from "follow and extract", but another extract :)

> Alternatively, we could name the option only “Extract”, replace “nothing” with “serp”, and consider the fact that following is needed for other types an implementation detail.

I think we want to make users aware of the crawling logic even more eventually, so I wouldn't consider it entirely an implementation detail.

What about "disabled"?